tests/run_pytest.py

```diff
@@ -1,42 +0,0 @@
-import sys
-import pytest
-import os.path
-from yaml import FullLoader, load as yaml_load
-
-
```

Comment:
Are you sure this file used nowhere else?
at least, it is used in `README.md`
Please add changes to `README.md`

Reply:
I have not found any files that apply this runner, including configurations, pipelines for CI.
@Alexander-Makaryev Am I right?

Reply:
at least, it is used in `README.md`
Please add changes to `README.md`